### PR TITLE
open-cluster-management/backlog#3430 ocp upg 3907

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: install/upgrade_ocp.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[#upgrading_ocp]
+= Upgrading {ocp-short}
+
+You can upgrade the version of {ocp} that hosts your {product-title} hub cluster. Backing up your data before initiating any cluster-wide upgrade is best practice.
```
Comment (dockerymick) [inline install/upgrade_ocp.adoc]: I suggest that the tone be more active: "Back up you data before you initiate any cluster-wide upgrade. 

Unless upgrade is optional?

File: install/upgrade_ocp.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+You can upgrade the version of {ocp} that hosts your {product-title} hub cluster. Backing up your data before initiating any cluster-wide upgrade is best practice. 
+
+During the upgrade of the {ocp-short} version, the {product-title-short} web console might show brief periods when pages or data are unavailable. Indicators can include HTTP 500 errors, HTTP 504 errors, or errors that data that was previously available is not available. This is a normal part of the upgrade, and no data is lost when this occurs. The availability will be restored. 
+
+The search index is also rebuilt during this upgrade, so any queries that are submitted during the upgrade might be incomplete.
+
+The following table contains some noted observations from an upgrade from {ocp-short} version 4.4.3 to 4.4.10:
+
+|===
```
Comment (dockerymick) [inline install/upgrade_ocp.adoc:12]: You can name the table with the following syntax in line 12 (add new line break): `.Title of the table`

File: install/upgrade_ocp.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+| Elapsed time (minutes:seconds) | Observed change | Duration
+
+| 03:40 | GRC UI experiences HTTP 500 | Service restored within 20 seconds
```
Comment (dockerymick) [inline install/upgrade_ocp.adoc]: Are "Observed changes" error messages? If not i suggest updating GRC UI to Governance and risk console .

File: install/upgrade_ver.adoc [hunk 4]
```diff
@@ -1,10 +1,9 @@
 [#upgrading]
 = Upgrading
 
-You can upgrade your Red Hat Advanced Cluster Management for Kubernetes operator from version 1.0 to version 1.0.1.
-When you installed Red Hat Advanced Cluster Management, you selected to enable Automatic upgrades or Manual upgrades.
+*Important:* Upgrading from the product technology preview version 1.x of {product-title} to the version 2.0.0 is not supported. You must uninstall the 1.x version and install version 2.0.0 as a clean install.
```
Comment (dockerymick) [inline install/upgrade_ver.adoc]: Change Upgrading to upgrade

Should you change 2.0.0 to 2.0 (i think Brandi mentioned that in one of my PRs)?

File: install/upgrade_ocp.adoc (hunk 3 above)
Comment (dockerymick) [reply install/upgrade_ocp.adoc]: If these are error messages, the addition sign on lines 17 and 18 should be fine, unless it is meant to indicate AND

**Review by dockerymick — APPROVED** [review]
Made a couple of comments, otherwise this looks good Chris!

File: install/upgrade_ocp.adoc (hunk 2 above)
Comment (cadawson) [reply install/upgrade_ocp.adoc:12]: Done

File: install/upgrade_ocp.adoc (hunk 1 above)
Comment (cadawson) [reply install/upgrade_ocp.adoc]: done

File: install/upgrade_ver.adoc (hunk 4 above)
Comment (cadawson) [reply install/upgrade_ver.adoc]: Left upgrading to have a verb, but changed 2.0. for consistency.

File: install/upgrade_ocp.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+The search index is also rebuilt during this upgrade, so any queries that are submitted during the upgrade might be incomplete.
+
+The following table contains some noted observations from an upgrade from {ocp-short} version 4.4.3 to 4.4.10:
+
+|===
+. Observations from an {ocp-short} upgrade from version 4.3.3 to 4.4.10.
+| Elapsed time (minutes:seconds) | Observed change | Duration
+
+| 03:40 | Gevernance and risk console experiences HTTP 500 | Service restored within 20 seconds
```
Comment (mdelder) [inline install/upgrade_ocp.adoc]: _Governance_

File: install/upgrade_ocp.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
+[#upgrading_ocp]
+= Upgrading {ocp-short}
+
+You can upgrade the version of {ocp} that hosts your {product-title} hub cluster. Back up your data before initiating any cluster-wide upgrade. 
+
+During the upgrade of the {ocp-short} version, the {product-title-short} web console might show brief periods when pages or data are unavailable. Indicators can include HTTP 500 errors, HTTP 504 errors, or errors that data that was previously available is not available. This is a normal part of the upgrade, and no data is lost when this occurs. The availability is eventually restored.
```
Comment (mdelder) [inline install/upgrade_ocp.adoc]: _HTTP 500 (Internal Server Error), HTTP 504 (Gateway Timeout Error)_

File: install/upgrade_ocp.adoc [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+During the upgrade of the {ocp-short} version, the {product-title-short} web console might show brief periods when pages or data are unavailable. Indicators can include HTTP 500 errors, HTTP 504 errors, or errors that data that was previously available is not available. This is a normal part of the upgrade, and no data is lost when this occurs. The availability is eventually restored. 
+
+The search index is also rebuilt during this upgrade, so any queries that are submitted during the upgrade might be incomplete.
+
+The following table contains some noted observations from an upgrade from {ocp-short} version 4.4.3 to 4.4.10:
+
+|===
+. Observations from an {ocp-short} upgrade from version 4.3.3 to 4.4.10.
+| Elapsed time (minutes:seconds) | Observed change | Duration
```
Comment (mdelder) [inline install/upgrade_ocp.adoc]: _Elapsed time of upgrade process_

.. want to find someway to clarify that it isn't the elapsed time that the service is unavailable. Maybe it's already clear?

File: install/upgrade_ver.adoc [hunk 8]
```diff
@@ -1,10 +1,9 @@
 [#upgrading]
 = Upgrading
 
-You can upgrade your Red Hat Advanced Cluster Management for Kubernetes operator from version 1.0 to version 1.0.1.
-When you installed Red Hat Advanced Cluster Management, you selected to enable Automatic upgrades or Manual upgrades.
+*Important:* Upgrading from the product technology preview version 1.x of {product-title} to version 2.0 is not supported. You must uninstall the 1.x version and install version 2.0 as a clean install.
```
Comment (mdelder) [inline install/upgrade_ver.adoc]: We need a topic on migration from 1.x to 2.x

**Review by mdelder — COMMENTED** [review]
Looks good! Minor comments within.

File: install/upgrade_ocp.adoc (hunk 5 above)
Comment (cadawson) [reply install/upgrade_ocp.adoc]: Thanks

File: install/upgrade_ocp.adoc (hunk 6 above)
Comment (cadawson) [reply install/upgrade_ocp.adoc]: Done

File: install/upgrade_ocp.adoc (hunk 7 above)
Comment (cadawson) [reply install/upgrade_ocp.adoc]: If you weren't sure, it can't hurt to add the extra clarification.

File: install/upgrade_ver.adoc (hunk 8 above)
Comment (cadawson) [reply install/upgrade_ver.adoc]: I modified the topic to be 1.x to 2.0 focused.  Does that work?

**Comment by cadawson** [comment]
Review comments provided, and content updated. Merging this PR to have the content for GA. If there are any issues, we can update it. Thanks!

**Pull request merged**